Library/Homebrew/manpages/brew.1.md

```diff
@@ -438,7 +438,8 @@ Note that these flags should only appear after a command.
     Display the file or directory used to cache <formula>.
 
   * `--cellar`:
-    Display Homebrew's Cellar path. *Default:* `/usr/local/Cellar`
+    Display Homebrew's Cellar path. *Default:* `$(brew --prefix)/Cellar`, or if
+    that directory doesn't exist, `$(brew --repository)/Cellar`.
```

Comment:
> If brew is being run from within a repository,

Note the "if", which refers to non-default situations.  In a default installation of Homebrew, the Cellar is /usr/local.  So that is what the man page should say.